app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -101,8 +101,12 @@ def live_output
       end
 
       def humanized_input
-        N_('%{description} on %{host}') % { :host => input[:host].try(:[], :name),
-          :description => input[:description].try(:capitalize) || input[:job_category] }
+        return unless input.present?
+
+        N_('%{description} on %{host}') % {
+          host: host.to_label,
```

Comment:
Do we _need_ to load the host here? Can't we pre-calculate this somewhere in `#plan` or `#inner_plan` where we already have the host loded?

Reply:
I'm not sure where can we add this in `#plan` or `#inner_plan`.
What about adding it here https://github.com/theforeman/foreman-tasks/blob/master/app/models/foreman_tasks/concerns/action_subject.rb#L14:
`hash.update(to_label: to_label) if respond_to? :to_label`

Reply:
I can't really tell from the top of my head what side effects adding it to ActionSubject might have. Adding it to the hash on L31-L34 in this file should do the trick